appveyor.yml

```diff
@@ -107,13 +107,6 @@ test_script:
 
 after_test:
  - cd "%WORK_DIR%" # Get back to where cache-s3.exe is located
```

Comment:
Why was caching removed? It's crucial, otherwise the build takes 80min instead of 15min :)

Reply:
Yeah I will zap that commit later but for now I need a green build from AppVeyor to test the Daedalus side. The AppVeyor build cruelly fails in the cache upload step!